src/Symfony/Component/Validator/Tests/Fixtures/Entity.php

```diff
@@ -19,7 +19,7 @@
  * @Assert\GroupSequence({"Foo", "Entity"})
  * @Assert\Callback({"Symfony\Component\Validator\Tests\Fixtures\CallbackClass", "callback"})
  */
-class Entity extends EntityParent
+class Entity extends EntityParent implements EntityInterface, EntityParentInterface
```

Comment:
I think you should add another interface like `OtherEntityInterface` (I don't really like the name but you got the point), because implementing both interfaces when on extends another does not make sense, this PR is not trying to fix that case.